torch/nn/modules/module.py

```diff
@@ -170,7 +170,8 @@ def apply(self, fn):
         Returns:
             Module: self
 
-        Example:
+        Example::
```

Comment:
When should we have `Example:` or `Example::`? I see that you have used `Example:` in other parts of this PR

Reply:
I think such blocks are called [literal blocks (fully minimized form)](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks). So they should all use double-colon really. But the main problem with this is that it is missing a newline.